### PR TITLE
Add "supported resources" section to all Ocean integration docs that still do not have it

### PR DESCRIPTION
**Pull request by lordsarcastic**
# Description

Some integration docs do not have "supported resources" section. This PR adds it to these integrations

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/jenkins/jenkins.md [hunk 1]
```diff
@@ -442,6 +442,14 @@ The integration makes use of the [JQ JSON processor](https://stedolan.github.io/
 
 The integration configuration determines which resources will be queried from Jenkins, and which entities and properties will be created in Port.
 
+:::tip Supported resources
+The following resources can be used to map data from Jenkins, it is possible to reference any field that appears in the API responses linked below for the mapping configuration.
+
+- `job` - `\<your-jenkins-host\>/api/json)
+- `build` - `\<your-jenkins-host\>/api/json)
```
Comment (PeyGis) [inline docs/build-your-software-catalog/sync-data-to-catalog/jenkins/jenkins.md:449]: Jenkins build and job are using the same API?

**Review by PeyGis — CHANGES_REQUESTED** [review]

File: docs/build-your-software-catalog/sync-data-to-catalog/jenkins/jenkins.md (hunk 1 above)
Comment (lordsarcastic) [reply docs/build-your-software-catalog/sync-data-to-catalog/jenkins/jenkins.md:449]: Yes, they are

Comment (lordsarcastic) [reply docs/build-your-software-catalog/sync-data-to-catalog/jenkins/jenkins.md:449]: Builds are found under jobs

File: docs/build-your-software-catalog/sync-data-to-catalog/jenkins/jenkins.md [hunk 2]
```diff
@@ -442,6 +442,14 @@ The integration makes use of the [JQ JSON processor](https://stedolan.github.io/
 
 The integration configuration determines which resources will be queried from Jenkins, and which entities and properties will be created in Port.
 
+:::tip Supported resources
+The following resources can be used to map data from Jenkins, it is possible to reference any field that appears in the API responses linked below for the mapping configuration.
+
+- `job` - (`\<your-jenkins-host\>/api/json`)
+- `build` - (`\<your-jenkins-host\>/api/json`)
+- `user` - (`\<your-jenkins-host\>/people/api/json`)
```
Comment (PeyGis) [inline docs/build-your-software-catalog/sync-data-to-catalog/jenkins/jenkins.md:450]:
```suggestion
- `job` - (`<your-jenkins-host>/api/json`)
- `build` - (`<your-jenkins-host>/api/json`)
- `user` - (`<your-jenkins-host>/people/api/json`)
```

**Review by PeyGis — CHANGES_REQUESTED** [review]

**Review by PeyGis — APPROVED** [review]
LGTM

**Review by hadar-co — APPROVED** [review]

**Pull request merged**